templates/kotlin/Project.cshtml

```diff
@@ -56,7 +56,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <None Update="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\$(TargetFramework);buildTransitive\$(TargetFramework)" />
+    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="@@(NuGetTargetFolders)" />
```

Comment:
I think you have a double `@@` here.

Reply:
The double `@@` is required because it's a Razor template and we have to escape it or else Razor treats it like a Razor variable.